c2corg_ui/static/js/addassociation.js

```diff
@@ -55,7 +55,25 @@ app.AddAssociationController = function(appApi, $attrs) {
   this.api_ = appApi;
 
 
+  /**
+   * @private
+   */
   this.dataset = $attrs['dataset'];
+
+  /**
+   * Typed directly in the directive HTML.
+   * It is the currently viewed document type.
+   *
+   * For example, if you are looking at a waypoint ->
+   * parentType = waypoint, parentID = waypoint's_id.
+   *
+   * The child type and ID will depend on the document selected
+   * in the results of the app-search dropdown.
+   * @private
+   */
+  this.parentDocType = $attrs['parentDocType'];
```

Comment:
Same comment here about the name and type.

To be honest I don't understand what your comment means.